### PR TITLE
Home values module

### PR DESCRIPTION
**Pull request by jgalobart**
Add values module at home

### Files
HTML: Resources/templates/responsive/home/partials/values.php
CSS: public/assets/sass/layouts/_home.scss
JS: public/assets/js/home/home.js
ICONS: public/assets/img/home

### Testing
1. Check if elements are well placed in desktop and mobile
2. Check if sliders are working
3. Check if project images are tinted
4. Check if footprints tab are working

### CODE REVIEW

**Review by javicarrillo — COMMENTED** [review]
Thanks,  we have two small improvements in mobile version:

- Add a line between the two modules (after SDGs), in the desktop version is a vertical line, in mobile it should be horizontal

- In the mobile version the link " Ver proyectos de huella ecológica " should be at the end of the module, after the projects slider.

**Comment by jgalobart** [comment]
@javicarrillo  

1. moved lateral border in mobile to bottom
2. fixed layout to show footprint link at last block in mobile. I need to avoid bootstrap rules and make my own grid to do that :(

File: public/assets/sass/layouts/_home.scss [hunk 1]
```diff
@@ -500,6 +500,292 @@ body.home {
...
+                a.active {
+                    filter: grayscale(0%);
+                }
+            }
+        }
+
+        div[id^="goteo-values-"] {
+            display:grid;
+            grid-template-columns: 60% 40%;
+        }
```
Comment (javicarrillo) [inline public/assets/sass/layouts/_home.scss]: This class breaks the bootstrap columns.

**Review by javicarrillo — COMMENTED** [review]
- Comment en 557 line
- Add hover arrows in sliders
- Add project links in projects slider
- The link "Ver proyectos de huella ecológica" is not at bottom in mobile version.

**Pull request closed without merging**